examples/enc_test.go

```diff
@@ -6,26 +6,26 @@ import (
 	"github.com/dedis/crypto/util/random"
 )
 
-func ElGamalEncrypt(suite crypto.Suite, pubkey crypto.Point, message []byte) (
+func ElGamalEncrypt(group crypto.Group, pubkey crypto.Point, message []byte) (
```

Comment:
Agreed.  In fact the Group interface is a fairly "suite-like" interface; we might discuss whether it should perhaps actually be renamed as such as well.  It's basically a suite interface for discrete-log groups.  (Note: I'm not suggesting at the moment we actually do this at the moment, but perhaps think about it.)